Microsoft.Toolkit.Uwp/Extensions/StringExtensions.cs

```diff
@@ -17,17 +18,39 @@ public static class StringExtensions
         /// Retrieves the provided resource for the current view context.
         /// </summary>
         /// <param name="resourceKey">Resource key to retrieve.</param>
+        /// <param name="uiContext">UIContext to use </param>
```

Comment:
We should use a see tag to link to the UIContext docs more easily. I assume this is something one would have in a XI context?

Reply:
UIContext is not specifically for XamlIslands (https://github.com/rkarman/AppWindowIntro#wait-theres-one-more-thing). Still, this is a terrible description. I'll update these.